lib/cartopy/crs.py

```diff
@@ -1454,7 +1454,7 @@ def __init__(self, central_longitude=0, globe=None):
         # 40 deg N introduced by incomplete fix to issue #113 (see
         # https://trac.osgeo.org/proj/ticket/113).
         import re
-        match = re.search(r"\d\.\d", PROJ4_RELEASE)
+        match = re.search(r"\d+\.\d+", PROJ4_RELEASE)
         if match is not None:
             proj4_version = float(match.group())
             if 4.8 <= proj4_version < 4.9:
```

Comment:
:+1: for switching to `\d+` (although in this specific case it's unlikely to cause a problem - it won't affect the subsequent version check until we hit 4.80 or greater), but this still isn't correct because of the float conversion: `float('4.80') == 4.8`.

How about factoring out the `PROJ4_RELEASE`-to-tuple code and sharing it with the azimuthal equidistant code? This comparison would then become `(4, 8, 0) <= proj4_version < (4, 9, 0)`.